build/phases.go

```diff
@@ -126,42 +121,37 @@ func (l *Lifecycle) newExport(repoName, runImage string, publish bool, launchCac
 				repoName,
 			),
 		)
-	} else {
-		if launchCacheName != "" {
-			return l.NewPhase(
-				"exporter",
-				WithDaemonAccess(),
-				WithArgs(
-					"-image", runImage,
-					"-layers", layersDir,
-					"-app", appDir,
-					"-daemon",
-					"-launch-cache", launchCacheDir,
-					repoName,
-				),
-				WithBinds(fmt.Sprintf("%s:%s", launchCacheName, launchCacheDir)),
-			)
-		} else {
-			return l.NewPhase(
-				"exporter",
-				WithDaemonAccess(),
-				WithArgs(
-					"-image", runImage,
-					"-layers", layersDir,
-					"-app", appDir,
-					"-daemon",
-					repoName,
-				),
-			)
-		}
 	}
+	if launchCacheName != "" {
```

Comment:
There are further simplifications we could make here. Looking at the code `launchCacheName` will never be an empty string so we probably don't need this guard/check.

Reply:
Thanks for the feedback. I removed some more code!